pkg/debug/start_debug.go

```diff
@@ -39,11 +39,13 @@ func StartDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alte
 }
 
 func startDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alternateImageValue string) error {
-	deployment, err := VerifyDeploymentExists(context, clusterNamespace, deploymentName)
+	OriginalDeployment, err := VerifyDeploymentExists(context, clusterNamespace, deploymentName)
 	if err != nil {
 		return fmt.Errorf("Missing mon or osd deployment name %s. %v\n", deploymentName, err)
 	}
 
+	deployment := *OriginalDeployment
```

Comment:
after these changes, I tested locally everything was working as expected.

Reply:
How about adding a comment that it is needed to dereference the var

Reply:
added